packages/snjs/lib/application.ts

```diff
@@ -773,8 +773,8 @@ export class SNApplication {
     return this.itemManager.findTagByTitle(title);
   }
 
-  public searchTags(searchQuery: string): SNTag[] {
-    return this.itemManager.searchTags(searchQuery);
+  public searchTags(searchQuery: string, note?: SNNote): SNTag[] {
```

Comment:
Hmm it might not be clear to a caller how nite is being used? I think I can take a good guess but maybe an @param comment doc would be helpful.

Reply:
Good call! I've improved the comment doc and added it to the application method too